bin/updateSquadsCategory.ts

```diff
@@ -0,0 +1,71 @@
+import fastq from 'fastq';
```

Comment:
Script to update existing Squads from this [JSON file](https://github.com/dailydotdev/daily-api/blob/e483f23e32dab8b22412c573d49b6f32f1475b02/bin/categorizedSquads.ts). Originally from a CSV [provided by product team](https://dailydotdev.slack.com/archives/C069XNYMDGV/p1726575033390519?thread_ts=1725703282.452569&cid=C069XNYMDGV).